src/components/bs5/callToAction/callToAction.hbs

```diff
@@ -0,0 +1,4 @@
+<a class="cta-link{{#ifCond styles '==' 'view all'}} view-all{{/ifCond}}{{#if sizeSmall}} small{{/if}}{{#if isdisabled}} disabled{{/if}}" {{#if isdisabled}}aria-disabled="true"{{/if}} href="{{href}}" target="{{target}}" {{#if arialabel}}aria-label="{{arialabel}}"{{/if}} {{{dataatts}}}>
```

Comment:
It would be good to reuse the lego block in https://github.com/qld-gov-au/qgds-bootstrap5/blob/develop/src/components/bs5/button/button.hbs instead of recreating the same element.  with same overrides.  
Or
Leverage how button in vanilla works.
https://github.com/qld-gov-au/qgds-vanilla/blob/main/src/components/_global/css/btn/stories/button.hbs